docs/index.md

```diff
@@ -13,7 +13,7 @@ title: Postgres.app – the easiest way to get started with PostgreSQL on the Ma
 
 <div class="buttons">
 	<a href="{{ site.downloadLocation }}" onclick="trackOutboundLink(this.href,'download');return false;" class="btn btn-primary btn-lg"><span class="glyphicon glyphicon-download-alt"></span> Download</a>
-	<a href="/documentation/" class="btn btn-default btn-lg"><span class="glyphicon glyphicon-book"></span> Documentation</a>
+	<a href="/docs/documentation/" class="btn btn-default btn-lg"><span class="glyphicon glyphicon-book"></span> Documentation</a>
```

Comment:
This change would break the link on the postgresapp.com website. The website is setup so that the /docs/ directory is at the root.